simapp/app.go

```diff
@@ -189,6 +189,17 @@ func NewSimApp(
 	var (
 		app        = &SimApp{}
 		appBuilder *runtime.AppBuilder
+		// Below we construct and set an application specific mempool and ABCI 1.0 Prepare and Process Proposal
+		// handlers.  These defaults are already set in the SDK's BaseApp, this shows an example of how to override
+		// them.
+		nonceMempool = mempool.NewNonceMempool()
+		mempoolOpt   = baseapp.SetMempool(nonceMempool)
+		prepareOpt   = func(app *baseapp.BaseApp) {
+			app.SetPrepareProposal(app.DefaultPrepareProposal())
+		}
+		processOpt = func(app *baseapp.BaseApp) {
+			app.SetProcessProposal(app.DefaultProcessProposal())
+		}
```

Comment:
can we assume if none is set we use default? Ideally this sort of thing only needs to be set if the app developer is doing something custom, otherwise its pointless boilerplate

Reply:
That is the current state of things, this useless boilerplate is just here to demonstrate *how* to set these to non-default values.

Reply:
ah okay, can we put a comment saying that or leave it commented out. We do the same below on some gov things, well we did in legacy.

Reply:
Is the current comment unclear? To me it is but I may have myopia at this point.

Reply:
I could comment it out too, yea.